documentation/content/codingstyle.rst

```diff
@@ -77,7 +77,7 @@ Placing Optional Bash Keywords and Spaces
 
 Some of the bash keywords may accept or not the reserved word `then` or `do`
 which can be added at the end of the expression or in the next line. In kw we
-put the then statement at the end of the expression (after the semicolon), as
+put the `then` statement at the end of the expression (after the semicolon), as
```

Comment:
Since you are already here it might be worth updating the placeholders used in the examples to the correct format of placing the text between angle brackets and using lower case letters. So `EXPRESSION` should be `<expression>` and so on.